zerver/lib/hotspots.py

```diff
@@ -64,9 +64,7 @@
 def get_next_hotspots(user: UserProfile) -> List[Dict[str, object]]:
     # For manual testing, it can be convenient to set
     # ALWAYS_SEND_ALL_HOTSPOTS=True in `zproject/dev_settings.py` to
-    # make it easy to click on all of the hotspots.  Note that
-    # ALWAYS_SEND_ALL_HOTSPOTS has some bugs; see ReadTheDocs (link
-    # above) for details.
+    # make it easy to click on all of the hotspots.
```

Comment:
Here is the demo of before/after fixing of the above bug.

|Before | After |
| --- | --- |
| ![before_fix](https://user-images.githubusercontent.com/63820270/117642634-2695be00-b1a5-11eb-8dcf-fd9354e42ff5.gif) |  ![after_fix](https://user-images.githubusercontent.com/63820270/117642619-21387380-b1a5-11eb-9137-4afbe257265d.gif) |